front/app/modules/commercial/insights/admin/containers/Insights/List/CreateInsightsView.tsx

```diff
@@ -124,21 +269,98 @@ export const CreateInsightsView = ({
           />
           {errors && <Error apiErrors={errors['name']} fieldName="view_name" />}
         </SectionField>
-        <SectionField>
-          <Select
-            id="view_project_scope"
-            options={getProjectOptions()}
-            onChange={onChangeProjectScope}
-            value={projectScope}
-            label={
-              <FormattedMessage {...messages.createModalProjectScopeLabel} />
-            }
-          />
-        </SectionField>
-        <Box display="flex" justifyContent="center">
+        <Box>
+          <Label>
+            <FormattedMessage {...messages.createModalProjectsLabel} />
+          </Label>
+          {!ideationProjects && <Spinner />}
+          {ideationProjects?.map((project) =>
+            project.attributes.folder_id ? null : (
+              <Box
+                key={project.id}
+                py="15px"
+                borderBottom={`1px solid ${colors.separation}`}
+              >
+                <Checkbox
+                  size="20px"
+                  onChange={() => toggleSelectProject(project.id)}
+                  label={localize(project.attributes.title_multiloc)}
+                  checked={selectedProjectsIds.includes(project.id)}
+                />
+              </Box>
+            )
+          )}
+          {folders.map((folder) => {
+            const isFolderExpanded = expandedFoldersIds.includes(folder.id);
+            return (
+              <Box key={folder.id}>
+                <Box
+                  py="15px"
+                  borderBottom={`1px solid ${colors.separation}`}
+                  display="flex"
+                  justifyContent="space-between"
+                  alignItems="center"
+                >
+                  <StyledCheckboxWithPartialCheck
+                    id={folder.id}
+                    size="20px"
+                    onChange={() => toggleSelectAllProjectsInFolder(folder)}
+                    label={folder.folderName}
+                    checked={isFolderSelected(folder)}
+                  />
+                  <Button
+                    onClick={() => toggleExpandFolder(folder.id)}
+                    buttonStyle="text"
+                    type="button"
+                    ariaExpanded={isFolderExpanded}
+                    py="0px"
+                  >
+                    {isFolderExpanded ? (
+                      <FormattedMessage {...messages.createModalCollapse} />
+                    ) : (
+                      <FormattedMessage {...messages.createModalExpand} />
```

Comment:
Very nitpicky but the 'Expand' text could be moved down a few pixels, now it's not completely aligned with the expand icon

![image](https://user-images.githubusercontent.com/13822005/153416151-44fed6b5-7ae3-443b-a8df-78ed660baaaa.png)

Reply:
Good that you spotted this! 👁️